### DISCUSSION TITLE
ReSpeaker 4-Mic Array for Raspberry Pi | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#718] [Q&A] [answered]
# ReSpeaker 4-Mic Array for Raspberry Pi | Seeed Studio Wiki

ReSpeaker 4-Mic Array for Raspberry Pi

https://wiki.seeedstudio.com/ReSpeaker_4_Mic_Array_for_Raspberry_Pi/

**Comment by DM10** [comment, marked as answer]
To use this with a uptodate Raspberry-Image see https://forum.seeedstudio.com/t/unable-to-setup-respeaker-on-raspberry-pi-4b/265601/4

Basically clone this repo: https://github.com/HinTak/seeed-voicecard

This info should be updated

**Reply by MatthewJeffson** [reply]
Thanks and the contents will be updated soon.
Best Regards,
Matthew
